internal/exec/exec.go

```diff
@@ -56,9 +56,7 @@ type Runner interface {
 	DescriptorProto(args []string) error
 	FieldDescriptorProto(args []string) error
 	ServiceDescriptorProto(args []string) error
-	Lint(args []string) error
-	ListLinters() error
-	ListAllLinters() error
+	Lint(args []string, listAllLinters bool, listLinters bool) error
```

Comment:
nit: `Lint(args []string, listAllLinters, listLinters bool) error`

Reply:
This actually is not the style of Prototool - I think it leads to serious errors that Golang does this, and I always specify fully. I did this in the GRPC function, but if it's coming up multiple times we should talk about it, and if you feel strongly, this should be consistent codebase-wide (where right now the only time the shorthand is used in the entire codebase is on GRPC).

Reply:
Sure, works for me.